migration-engine/migration-engine-tests/tests/migrations/mysql.rs

```diff
@@ -217,8 +217,18 @@ fn native_type_columns_can_be_created(api: TestApi) {
         ("decimal", "Decimal", "Decimal(5, 3)", "decimal(5,3)"),
         ("float", "Float", "Float", "float"),
         ("double", "Float", "Double", "double"),
-        ("bits", "Bytes", "Bit(10)", "bit(10)"),
-        ("bit", "Boolean", "Bit(1)", "bit(1)"),
+        (
+            "bits",
+            "Bytes",
+            "Bit(10)",
+            if api.is_tidb() { "bit(10) unsigned" } else { "bit(10)" },
+        ),
+        (
+            "bit",
+            "Boolean",
+            "Bit(1)",
+            if api.is_tidb() { "bit(1) unsigned" } else { "bit(1)" },
```

Comment:
Are these type differences in TiDB documented somewhere?

Reply:
Related issue: https://github.com/pingcap/tidb/issues/25472

I contacted the R&D engineer (@hawkingrei) in charge of the issue, who will investigate and handle the issue later.

We can temporarily adjust the test case to make it work. If the issue is closed, I will resubmit a PR for modification or further explain the problem.

Reply:
@hawkingrei will this be released with the next minor server release 6.4 ?

Reply:
Should probably also be in nightly now, so a PR that replaces the Docker image with nightly and undoes this change above should fix it?